React/Views/RCTFont.mm

```diff
@@ -62,13 +63,14 @@ static RCTFontWeight weightOfFont(UIFont *font)
       @(UIFontWeightRegular),
       @(UIFontWeightMedium),
       @(UIFontWeightSemibold),
+      @(UIFontWeightSemibold),
       @(UIFontWeightBold),
       @(UIFontWeightHeavy),
       @(UIFontWeightBlack)
     ];
   });
 
-  for (NSInteger i = 0; i < fontNames.count; i++) {
+  for (NSUInteger i = 0; i < fontNames.count; i++) {
```

Comment:
The modern Apple convention is "always use `NSInteger` unless you really need to use `NSUInteger`", we are following it in RN.